src/modules/chuck-norris/categories.ts

```diff
@@ -0,0 +1,42 @@
+import { InternetModule } from "@faker-js/faker"
+
+const facts = require("./api/chuckfacts.json")
+
+// const array = [{
```

Comment:
Can we remove these parts of commented out code please :)

Reply:
can do